nixio/multi_tag.py

```diff
@@ -52,7 +52,14 @@ def positions(self, da):
             raise TypeError("MultiTag.positions cannot be None.")
         if "positions" in self._h5group:
             del self._h5group["positions"]
-        self._h5group.create_link(da, "positions")
+        pos = da
+        if not isinstance(da, DataArray):
+            blk = self._parent
+            name = "f{self.name}-positions"
+            if name in blk.data_arrays:
+                del blk.data_arrays[name]
+            pos = blk.create_data_array(name, "multi_tag_positions", data=da)
```

Comment:
Might be nicer if the type follows the same format as the naming: same type as the parent but with `"-positions"` or `".positions"` appended.